### DISCUSSION TITLE
BigDecimal.setScale() returns detekt exception

### DISCUSSION THREAD

**Discussion by thiagomkuhl** [detekt/detekt#6350] [General]
Hello,

I’m with an issue when I try to use BigDecimal.setScale. 

The detekt always fail’s and returns that message:

“The original exception message was: For input string: “-1, 0, 1, 2” 

Somebody with that problem?

**Comment by cortinico** [comment]
Could you provide the code snippet? I have the feeling what you're reporting is a `MagicNumber` violation but I'm not entirely sure
